uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/shared/components/leave-request-popup-files-tab.js

```diff
@@ -4,25 +4,30 @@ define([
   'common/lodash',
   'common/moment',
   'leave-absences/shared/modules/components',
+  'common/services/file-upload',
+  'common/services/file-mime-types',
   'common/services/hr-settings'
 ], function (_, moment, components) {
   components.component('leaveRequestPopupFilesTab', {
     bindings: {
       canManage: '<',
+      fileUploader: '=',
```

Comment:
Because we are updating the object in files-tab which is used in request-ctrl. So it should be 2 way.